### ISSUE TITLE
Crashing after 16.1.1 update (0x20f296200)

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero-ios#587]
https://forums.zotero.org/discussion/101333/zotero-crashing-after-ios-16-1-1-update

Works fine for me on 16.1.1.

```
Exception Type:  SIGABRT
Exception Codes: #0 at 0x20f296200
Crashed Thread:  1

Thread 9 Crashed:
0   libsystem_kernel.dylib              0x000000020f296200 0x20f28f000 + 29184
1   libsystem_c.dylib                   0x00000001da589c8c 0x1da569000 + 134284
2   Zotero                              0x000000010505b2c4 0x1045b4000 + 11170500
3   Zotero                              0x000000010505b604 0x1045b4000 + 11171332
4   Zotero                              0x000000010505b380 0x1045b4000 + 11170688
5   Zotero                              0x0000000104c55f28 0x1045b4000 + 6954792
6   Zotero                              0x0000000104deab60 0x1045b4000 + 8612704
7   Zotero                              0x0000000104df42e0 0x1045b4000 + 8651488
8   Zotero                              0x0000000104e0798c 0x1045b4000 + 8731020
9   Zotero                              0x0000000104e16000 0x1045b4000 + 8790016
10  Zotero                              0x0000000104e1273c 0x1045b4000 + 8775484
11  Zotero                              0x0000000104e1512c 0x1045b4000 + 8786220
12  Zotero                              0x0000000105015e10 0x1045b4000 + 10886672
13  Zotero                              0x0000000105021d8c 0x1045b4000 + 10935692
14  Zotero                              0x0000000104e70c38 0x1045b4000 + 9161784
15  Zotero                              0x0000000104c5509c 0x1045b4000 + 6951068
16  Zotero                              0x0000000104db4334 0x1045b4000 + 8389428
17  Zotero                              0x00000001047d0a00 0x1045b4000 + 2214400
18  Zotero                              0x00000001047fc408 0x1045b4000 + 2393096
19  Zotero                              0x0000000104632ccc 0x1045b4000 + 519372
20  Zotero                              0x0000000104db2edc 0x1045b4000 + 8384220
21  Zotero                              0x000000010460f990 0x1045b4000 + 375184
22  Zotero                              0x00000001048a53f8 0x1045b4000 + 3085304
23  Zotero                              0x00000001047628b0 0x1045b4000 + 1763504
24  Zotero                              0x0000000104a8f584 0x1045b4000 + 5092740
25  Zotero                              0x0000000104a48afc 0x1045b4000 + 4803324
26  Zotero                              0x0000000104a79ce8 0x1045b4000 + 5004520
27  Zotero                              0x0000000104a793b0 0x1045b4000 + 5002160
28  Zotero                              0x0000000104a792b0 0x1045b4000 + 5001904
29  Zotero                              0x0000000104a6eb20 0x1045b4000 + 4959008
30  Zotero                              0x0000000104a9399c 0x1045b4000 + 5110172
31  Zotero                              0x0000000104a52ac4 0x1045b4000 + 4844228
32  Zotero                              0x0000000104a483f8 0x1045b4000 + 4801528
33  libdispatch.dylib                   0x00000001da5244b4 0x1da522000 + 9396
34  libdispatch.dylib                   0x00000001da525fdc 0x1da522000 + 16348
35  libdispatch.dylib                   0x00000001da52d694 0x1da522000 + 46740
36  libdispatch.dylib                   0x00000001da52e214 0x1da522000 + 49684
37  libdispatch.dylib                   0x00000001da52d56c 0x1da522000 + 46444
38  libdispatch.dylib                   0x00000001da52e1e0 0x1da522000 + 49632
39  libdispatch.dylib                   0x00000001da538e10 0x1da522000 + 93712
40  libsystem_pthread.dylib             0x000000021f632df8 0x21f632000 + 3576
41  libsystem_pthread.dylib             0x000000021f632b98 0x21f632000 + 2968
```

**Comment by michalrentka** [comment]
This should be fixed by Realm update for which we're waiting. Their bug can cause database corruption and it's non-reversible unfortunately.

**Comment by michalrentka** [comment]
Should be fixed in 66ac152a28fb5d5f294e5fdc2bd65605a258b365, affected users have to re-install unfortunately, but we shouldn't see new cases of this.
